A66-otel-stats.md

```diff
@@ -0,0 +1,325 @@
+    *Attributes*: grpc.method, grpc.target, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+
+#### Client Per-Call Instruments
+
+*   **grpc.client.call.duration** <br>
+    This metric aims to measure the end-to-end time the gRPC library takes to complete an RPC from the application’s perspective. <br>
+    Start timestamp - After the client application starts the RPC. <br>
+    End timestamp - Before the status of the RPC is delivered to the application. <br>
+    If the implementation uses an interceptor then the exact start and end timestamps would depend on the ordering of the interceptors. Non-interceptor implementations should record the timestamps as close as possible to the top of the gRPC stack, i.e., payload serialization should be included in the measurement. <br>
+    *Attributes*: grpc.method, grpc.target, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+#### Server Instruments
+
+*   **grpc.server.call.started** <br>
+    The total number of RPCs started, including those that have not completed. <br>
+    *Attributes*: grpc.method, grpc.authority <br>
+    *Type*: counter <br>
+    *Unit*: {call} <br>
+*   **grpc.server.call.sent_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) sent across all response messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.rcvd_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) received across all request messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.duration** <br>
+    This metric aims to measure the end2end time an RPC takes from the server transport’s (HTTP2/ inproc / cronet) perspective. <br>
+    Start timestamp - After the transport knows that it's got a new stream. For HTTP2, this would be after the first header frame for the stream has been received and decoded. Whether the timestamp is recorded before or after HPACK is left to the implementation. <br>
+    End timestamp - Ends at the first point where the transport considers the stream done. For HTTP2, this would be when scheduling a trailing header with END_STREAM to be written, or RST_STREAM, or a connection abort. Note that this wouldn’t necessarily mean that the bytes have also been immediately scheduled to be written by TCP. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+## OpenTelemetry Plugin Architecture
+
+To-be-filled
+
+This section describes a CallTracer approach to collect the client and server
```

Comment:
We should probably describe the CallTracer API in more detail, so that its nature is clear to the reader.  Obviously, the details will be language-specific, but the semantics should be similar enough across languages to allow a common description.